05/app.js

```diff
@@ -34,4 +34,37 @@ function insertUsers(usersList) {
 
         ulElement.appendChild(liElement);
     });
+    submit();
 }
+
+function submit() {
+    const btn = document.querySelector('.form__submit');
```

Comment:
Zazwyczaj warto wyszukać elementy na początku, aby tego nie robić za każdym razem (to się też tyczy poprzednich rozwiązań)